docs_manual/source/metadata_advanced.rst

```diff
@@ -182,3 +183,18 @@ Protein Expression Profiling / Mass Spectrometry Plugin
     * Files are linked to ``RawData`` under the assay.
 - Used with measurement type / technology type
     * protein expression profiling / mass spectrometry
+
+Protein Expression Profiling / Mass Cytometry Plugin
+-------------------------------------------------------
+
+- Internal name: ``samplesheets_assay_cytof``
+- Additional assay shortcuts
+    * N/A
+- Row-specific links
+    * Each row links to the **Assay Name** of the **mass cytometry** process node of that row.
```

Comment:
Line length over 80 here, please shorten. Other than these two comments, looks good and can merge.